integration_tests/test_snap.py

```diff
@@ -105,6 +105,19 @@ def test_snap_directory(self):
         self.run_snapcraft(['snap', 'prime'])
         self.assertThat(snap_file_path, FileExists())
 
+    def test_assemble_directory(self):
+        self.copy_project_to_cwd('assemble')
+        self.run_snapcraft('snap')
+
+        snap_file_path = 'assemble_1.0_{}.snap'.format(self.deb_arch)
+        os.remove(snap_file_path)
+
+        # Verify that Snapcraft can snap its own snap directory (this will make
+        # sure `snapcraft snap` and `snapcraft snap <directory>` are always in
```

Comment:
This should probably say `snapcraft assemble <directory>`?

Reply:
done